articles/application-insights/app-insights-ip-addresses.md

```diff
@@ -199,3 +199,12 @@ Open ports 80 (http) and 443 (https) for incoming traffic from these addresses:
 | --- | --- | --- | --- |
 | Agent | agent.azureserviceprofiler.net | dynamic | 443
 | Portal | gateway.azureserviceprofiler.net | dynamic | 443
+| Storage | core.windows.net | dynamic | 443
```

Comment:
should it say *.core.windows.net to be more specific? Or maybe you can specify which storage account? This looks super generic